src/histogram/histograms.rs

```diff
@@ -78,6 +78,20 @@ impl<A: Ord> Histogram<A> {
     pub fn grid(&self) -> &Grid<A> {
         &self.grid
     }
+
+    /// Returns the cumulative distribution function of a histogram. 
+    /// Equivalent to the numpy histogram function cumsum
```

Comment:
I'd like to have a more detailed docstring here: it should include the math formula to compute the value indexed `i_1`, ..., `i_n` in the final array and a simple example using a 1d/2d array.

Given that you are mentioning NumPy, it would be good to link directly to the docs for `np.cumsum`.